src/components/buttons/RegularButton.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Filenames in this project always use kebab case names, so `regular-button.strories.tsx` and not camel case.

For consistency this should be used on all new files. In general it's a matter of personal taste, although I find using kebab case names more simple to read when working in the console (not sure if I this theory can be baked by real science :D)